src/loader/mod.rs

```diff
@@ -81,6 +80,10 @@ pub enum Error {
     ReadBzImageHeader,
     /// Unable to read bzImage compressed image.
     ReadBzImageCompressedKernel,
+    /// Unable to read image magic number.
+    ReadImageMagicNumber,
+    /// Unable to read entry address.
+    ReadEntryAddress,
```

Comment:
Same obs. as above: I'm thinking of debugging possibilities for a kernel loading issue.

Reply:
Removed both. But I added a new one `InvalidPeImage` for common parsing error.